magpie-run-phoenix-performanceeval

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#############################################################################
+#  Copyright (C) 2013-2015 Lawrence Livermore National Security, LLC.
+#  Produced at Lawrence Livermore National Laboratory (cf, DISCLAIMER).
+#  Written by Albert Chu <chu11@llnl.gov>
+#  LLNL-CODE-644248
+#  
+#  This file is part of Magpie, scripts for running Hadoop on
+#  traditional HPC systems.  For details, see <URL>.
+#  
+#  Magpie is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  
+#  Magpie is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with Magpie.  If not, see <http://www.gnu.org/licenses/>.
+#############################################################################
+
+# This script is for running the pig testpig sanity test.  For the
+# most part, it shouldn't be editted.  See job submission files for
+# configuration details.
+
+source ${MAGPIE_SCRIPTS_HOME}/magpie-common-exports
+source ${MAGPIE_SCRIPTS_HOME}/magpie-common-functions
+source ${MAGPIE_SCRIPTS_HOME}/magpie-variable-conversion
+
+# For this run, we will use cluster specific paths
+Magpie_make_all_local_dirs_node_specific
+
+if [ "${PHOENIX_PERFORMANCEEVAL_ROW_COUNT}X" == "X" ]
+then
+    phoenixrowcount=100000
+else
+    phoenixrowcount=$PHOENIX_PERFORMANCEEVAL_ROW_COUNT
+fi
+
+phoenixquorumnodes="$(grep -A1 quorum $HBASE_CONF_DIR/hbase-site.xml | tail -n1 | cut -d ">" -f2 | cut -d "<" -f1)"
+zookeepernode="$(echo ${phoenixquorumnodes} | cut -d "," -f1)"
+
+# Phoenix's test relies on HBASE_CONF_DIR being set so make sure its from a region server
+regionserver=$(cat $HBASE_CONF_DIR/regionservers | head -n 1)
+export HBASE_CONF_DIR=$(echo ${PHOENIX_CONF_DIR} | sed "s/$(hostname)/$regionserver/g")
+
```

Comment:
I'm a little confused what's going on here.  Why would HBASE_CONF_DIR have to be modified?